eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Delegs.hs

```diff
@@ -169,14 +178,16 @@ conwayDelegsTransition = do
       -- It is impossible to have 65535 number of certificates in a
       -- transaction, therefore partial function is justified.
       let ptr = Ptr slot txIx (mkCertIxPartial $ toInteger $ length gamma)
-      trans @(EraRule "CERT" era) $
+      newState <- trans @(EraRule "CERT" era) $
         TRC (DelplEnv slot ptr pp acnt, certState', c)
+      trans @(EraRule "DELEG" era) $
+        TRC (DelegEnv slot ptr acnt pp, newState, c)
```

Comment:
This is supposed to be called from the CERT rule. It would make much more sense to call DELEGS rule CERTS rule instead, but this can be left for another PR.

Reply:
Nice suggestion by the way! :+1: :smiley: